examples/react-gatsby/src/components/layout.css

```diff
@@ -6,3 +6,14 @@ html {
     "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji";
   margin: 0;
 }
+
+.Layout {
+  display: flex;
+  flex-direction: column;
+  height: 100vh;
+}
+
+.MainContent {
```

Comment:
These are called in 3 different ways across the examples (`.MainContent`, `.main-content`, `main`). Can we maybe unify it?

Reply:
I was just using the conventions that I already encountered in each of the projects :P

Sure, I'll try to unify that, but I don't believe that we have to keep those unified. They are literally different apps. They have even completed different headers. :D

Reply:
Good point. 👍